apps/api-extractor/src/api/IConfigFile.ts

```diff
@@ -405,4 +406,15 @@ export interface IConfigFile {
    * to prevent spurious diffs in output files tracked for tests.
    */
   testMode?: boolean;
+
+  /**
+   * Specifies how API Extractor sorts members of an enum when generating api.json, api.md and rollup.d.ts.
+   * The default behaviour has been alphabetical, ByName in this case.
```

Comment:
Consider removing "The default behaviour has been alphabetical, ByName in this case" as it's redundant to `@defaultValue`.